src/main/java/org/sobotics/guttenberg/clients/Guttenberg.java

```diff
@@ -30,6 +30,23 @@
  * Fetches and analyzes the data from the API
  * */
 public class Guttenberg {	
+	
+	//Global Statistics
+	/**
+	 * The time when the last check was finished
+	 * */
+	public static Instant lastExecutionFinished = null;
+	
+	/**
+	 * The number of new answers that have been checked
+	 * */
+	public static int numberOfCheckedTargets = 0;
```

Comment:
Problem with those static global fields, is that it breaks when ran in parallel. `++` isn't synchronized.